Include/cpython/pyatomic.h

```diff
@@ -321,6 +321,27 @@ _Py_atomic_load_ptr(const void *obj);
 static inline int
 _Py_atomic_load_int_relaxed(const int *obj);
 
+static inline char
+_Py_atomic_load_char_relaxed(const char *obj);
```

Comment:
I know this is not in the scope of this PR, but those additions make me feel that all this code should be auto-generated by some new clinic magic for instance. One reason is that it's easy to miss a type when C/C.

Reply:
Totally agree.

It's exactly what happened here: https://github.com/python/cpython/pull/123211/commits/50a7bb61adc5c6f00f627514f6729f20289778c7